Jenkinsfile

```diff
@@ -1163,6 +1163,11 @@ def loadConfigEnvVars(){
 
   // Skip all the stages for changes only related to the documentation
   env.ONLY_DOCS = isDocChangedOnly()
+
+  // Run the ITs by running only if the changeset affects a specific module.
+  // For such, it's required to look for changes under the module folder and exclude anything else
+  // such as ascidoc and png files.
+  env.MODULE = getGitMatchingGroup(pattern: '[a-z0-9]+beat\\/module\\/([^\\/]+)\\/.*', exclude: '^(((?!\\/module\\/).)*$|.*\\.asciidoc|.*\\.png)')
```

Comment:
Given that we're now excluding anything that's not under `/module/`, do we also need the `.*\.asciidoc` and `.*\.png` patterns in the exclude list?

Reply:
Yes, we do, otherwise all those PRs that change the top-level `CHANGELOG.next.asciidoc` will not match the pattern, besides, the exclude does include the `png` files similar to the pattern that it is used  for whether the PR is only related to docs changes. We could potentially remove the png if required, althogh I don't know all the cases, but the asciidoc is needed.

Reply:
Thanks, lets keep this for now and optimize later.